lib/ukbitops/Makefile.uk

```diff
@@ -0,0 +1,20 @@
+$(eval $(call addlib_s,libukbitops,$(CONFIG_LIBUKBITOPS)))
+
+CINCLUDES-$(CONFIG_LIBUKBITOPS)	 		+= -I$(LIBUKBITOPS_BASE)/include
+ASINCLUDES-$(CONFIG_LIBUKBITOPS)	  	+= -I$(LIBUKBITOPS_BASE)/include
```

Comment:
Similarly to `atomic.h`, I doubt anything defined in `bitops.h` can be used by asm code, so all the `ASINCLUDES` in this file can go away.